### DISCUSSION TITLE
Einbinder (1975)

### DISCUSSION THREAD

**Discussion by iandoug** [precondition/precondition.github.io#18] [Show and tell]
![einbinder-1975-mod-ian en matrix](https://user-images.githubusercontent.com/12673886/116615640-e3815280-a93b-11eb-8f91-27ab6baa6c1a.jpg)
![einbinder-orthogonal en ergodox](https://user-images.githubusercontent.com/12673886/116615646-e4b27f80-a93b-11eb-9bf8-5eb368a6ef62.jpg)

**Comment by precondition** [comment]
[Added](https://github.com/precondition/precondition.github.io/commit/d96fafce8f7fff5c8d8d60df3278908ae49517a3)
